src/clr-angular/button/_properties.buttons.scss

```diff
@@ -4,8 +4,7 @@
 
 @include exports('button.properties') {
   @if $clr-use-custom-properties == true {
-    :root,
-    :host {
+    :root {
```

Comment:
Do you remember why we did both `:root` and `:host`? I vaguely remember adding this for a specific use case (was it browser support?) And why do we not need it anymore?

Reply:
So we originally had both for two reasons. First root was for clr-ui since everything is global. Root works for some cases with the web component but if in the web component we refer and try to reassign a var it can break down.

```scss
:host(state='danger') {
  --background: red; // this override is scoped to :host not :root first
}
```

So when we wanted to change the var based on the component state global vars dont work very well. So we added both `:root` and `:host` to the selector so all properties would apply to both global `:root` scope and local `:host` scope. This actually after debugging got a bit weird since it was sometimes hard to tell which property was local or global in the dev tools.

This worked but after refactoring to follow the 'modify var locally based on component state pattern' like (badges/tags) it didn't seem like we even needed most of the stateful global clr-ui vars in the core component. The component could have all its vars scoped directly to its `:host`. There was no need to have the duplicate selector and pull in all those vars when the component only used a small subset of vars that clr-ui used.